src/projectBank2/Account.java

```diff
@@ -0,0 +1,77 @@
+package projectBank2;
+
+import java.math.BigDecimal;
+
+public class Account {
+
+    private BigDecimal myPLN;

```

Comment:
Replace all myXXX with one field e.g value. Normally you have client and provider, the way you did it forces client to change the provider (Account class) if new you account needs to be added. Hope that is clear